src/styles/snake/_snake.scss

```
@@ -0,0 +1,18 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  @include screen(750px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  @include screen(450px) {
+    grid-template-columns: 1fr;
+  }
```

So there is easier solution to this one using grid's instruments
And you will need only one media query
```suggestion
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax($box-size, 1fr));
  
  @media (min-width: 1200px) {
    & {
      grid-template-columns: repeat(3, 1fr);
    }
```